MSAL/src/native_auth/public/state_machine/delegate_dispatcher/SignInDelegateDispatchers.swift

```diff
@@ -77,6 +77,21 @@ final class SignInStartDelegateDispatcher: DelegateDispatcher<SignInStartDelegat
         }
     }
 
+    func dispatchJITRequired(authMethods: [MSALAuthMethod], newState: RegisterStrongAuthState, correlationId: UUID) async {
+        if let onSignInJITRequired = delegate.onSignInStrongAuthMethodRegistration {
+            telemetryUpdate?(.success(()))
+            await onSignInJITRequired(authMethods, newState)
+        } else {
+            let error = SignInStartError(
+                type: .generalError,
+                message: requiredErrorMessage(for: "onSignInJITRequired"),
```

Comment:
I don't think this is the right name of the method "onSignInJITRequired". It should be:
```suggestion
                message: requiredErrorMessage(for: "onSignInStrongAuthMethodRegistration"),
```